src/KOKKOS/fix_rx_kokkos.cpp

```diff
@@ -1389,9 +1389,9 @@ void FixRxKokkos<DeviceType>::operator()(Tag_FixRxKokkos_solveSystems<ZERO_RATES
     // Store the solution back in dvector.
     for (int ispecies = 0; ispecies < nspecies; ispecies++)
     {
-      if (y[ispecies] < -MY_EPSILON)
+      if (y[ispecies] < -1.0e-10)
```

Comment:
Yep. Will do.  The key issues as I recall is that DBL_EPSILON is too small and will often throw.